.storybook/theme.js

```diff
@@ -0,0 +1,5 @@
+
+      import { create } from '@storybook/theming/create';
+
+      export default create({"base":"light","colorPrimary":"#607DBE","colorSecondary":"#F07D61","brandTitle":"@biotope/preview","brandUrl":"https://biotope.sh/","brandImage":"https://biotope.sh/_assets/biotope-logo.svg"});
```

Comment:
Can you please exclude this file from the commit? 🙂 It's auto-generated on preview build.